modules/gapi/include/opencv2/gapi/render.hpp

```diff
@@ -0,0 +1,53 @@
+#ifndef OPENCV_GAPI_RENDER_HPP
+#define OPENCV_GAPI_RENDER_HPP
+
+#include <string>
+#include <vector>
+
+#include <opencv2/gapi/opencv_includes.hpp>
+#include <opencv2/gapi/util/variant.hpp>
+#include <opencv2/gapi/own/exports.hpp>
+#include <opencv2/gapi/own/scalar.hpp>
+
+namespace cv
+{
+namespace gapi
+{
+
+struct TextEvent
```

Comment:
Y user should care about events, etc.

Let it be just
```
cv::gapi::draw::Text{ text, ... }
```

Reply:
Fixed